### ISSUE TITLE
Streaming mp3 and wav files

### ISSUE THREAD

**Issue by waltcapers** [kokorin/Jaffree#107]
This is not so much of an issue but a question.  I am new to Jaffree and ffmpeg and am trying to use it to stream mp3 and wav files to an rtmp server and also adjust the volume.  Here is what I am trying to do

                    FFmpeg.atPath(Paths.get(ffmpegPath))
                            .addInput(
                                    UrlInput
                                            .fromUrl(mp3_file)
                                            .addArgument("-re")
                            )
                            .addOutput(
                                    UrlOutput
                                            .toUrl(uri)
                                            .setFormat("flv")
                                            .copyAllCodecs()
                            )
                            .setOverwriteOutput(true)
                            .execute();

Basically, add  -filter:a "volume=.1".  Any suggestions.

**Comment by kokorin** [comment]
Filtering is not possible while copying codec. Remove `copyAllCodecs()` and add filtering.
